satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -1351,19 +1469,22 @@ datasets:
     resolution: 32000
     file_type: nc_fci_asr
     file_key: quality_reflectance
-    long_name: reflectance_quality
     fill_value: -1
+    long_name: TOA Bidirectional Reflectance % confidence
+    standard_name: reflectance_quality
     coordinates:
     - longitude
     - latitude
 
   quality_radiance:
     name: quality_radiance
     resolution: 32000
+    wavelength: []
```

Comment:
please remove.

Reply:
fixed
I had a few of those because if I don't include that line in the script, the writing of the full bloc is messed up somehow. So I had to remove it manually afterward.